test/camunda-platform/OrderExtensionElementsBehaviorSpec.js

```diff
@@ -0,0 +1,100 @@
+import {
+  bootstrapCamundaPlatformModeler,
+  inject
+} from 'test/TestHelper';
+
+import {
+  getBusinessObject
+} from 'bpmn-js/lib/util/ModelUtil';
+
+import diagramXML from './camunda-call-activity.bpmn';
+
+/* global sinon */
+
+describe('OrderExtensionElementsBehavior', function() {
+
+  beforeEach(bootstrapCamundaPlatformModeler(diagramXML));
+
+  it('should reorder extension elements',
+    inject(function(elementRegistry, modeling, bpmnFactory) {
+
+      // given
+      const callActivity = elementRegistry.get('CallActivity_1');
+      const businessObject = getBusinessObject(callActivity);
+
+      // when
+      const extensionElements = businessObject.get('extensionElements');
+
+      const ioElements = [
+        { type: 'camunda:Out', target: 'foo', source: 'out' },
+        { type: 'camunda:In', target: 'bar', source: 'in' },
+        { type: 'camunda:Out', variables: 'all' },
+        { type: 'camunda:In', variables: 'all' }
+      ];
+
+      addExtensionElements(extensionElements, bpmnFactory, ioElements);
+
+      modeling.updateModdleProperties(
+        callActivity,
+        extensionElements,
+        { values: extensionElements.get('values') }
+      );
+
+      // then
+      const updatedExtensionElements = businessObject.get('extensionElements').get('values');
+
+      expect(updatedExtensionElements[0]).to.have.property('variables', 'all');
+      expect(updatedExtensionElements[0]).to.have.property('$type', 'camunda:In');
```

Comment:
Let's use standard bpmn-js utils:
```suggestion
      expect(is(updatedExtensionElements[0], 'camunda:In'), 'should be camunda:In').to.be.true;
```